code/obj/machinery/dispenser.dm

```diff
@@ -83,6 +84,15 @@
 		ui = new(user, src, "TankDispenser", name)
 		ui.open()
 
+/obj/machinery/dispenser/ui_state(mob/user)
+	return tgui_physical_state
+
+/obj/machinery/dispenser/ui_status(mob/user)
+  return min(
+		tgui_physical_state.can_use_topic(src, user),
+		tgui_not_incapacitated_state.can_use_topic(src, user)
+	)
```

Comment:
Rather than overriding `ui_state` for the specific machine, I would consider adding a check based on the `status` flag for `REQ_PHYSICAL_ACCESS` on `/obj/machinery/ui_state` instead. This would mean the remote access is consistent based on the flag, rather than having to implement this on every machine that should be physical-access-only, which is just asking for them to get out of sync.